tests/test_e2e.py

```diff
@@ -0,0 +1,47 @@
+import neptune.new as neptune
+import torch
+from neptune.new.integrations.sacred import NeptuneObserver
+
+
+def test_e2e(experiment, model):
+    if torch.device("cuda:0"):
```

Comment:
Why do we need it? We're not running it on GPU on CI/CD. Is it for local testing?